forest/src/cli/config.rs

```diff
@@ -34,9 +35,10 @@ pub struct Config {
 
 impl Default for Config {
     fn default() -> Self {
+        let dir = ProjectDirs::from("com", "ChainSafe", "Forest").expect("failed to find project directories, please set FOREST_CONFIG_PATH environment variable manually.");
```

Comment:
Ideally we would factor this out so we only have a single source of truth for the ProjectDirs. But that can wait until another PR.